pkg/skaffold/build/cluster/kaniko_test.go

```diff
@@ -104,12 +104,20 @@ func TestArgs(t *testing.T) {
 			},
 			shouldErr: true,
 		},
+		{
+			description: "skip tls",
+			artifact: &latest.KanikoArtifact{
+				DockerfilePath: "Dockerfile",
+				SkipTLS:        true,
+			},
+			expectedArgs: []string{"--skip-tls-verify-registry", "gcr.io"},
+		},
 	}
 	for _, test := range tests {
 		testutil.Run(t, test.description, func(t *testutil.T) {
-			commonArgs := []string{"--dockerfile", "Dockerfile", "--context", "context", "--destination", "tag", "-v", "info"}
+			commonArgs := []string{"--dockerfile", "Dockerfile", "--context", "context", "--destination", "gcr.io/tag", "-v", "info"}
 
-			args, err := args(test.artifact, "context", "tag")
+			args, err := args(test.artifact, "context", "gcr.io/tag")
```

Comment:
not a skaffold issue, but for my understanding, kaniko builds one destination image so passing registry here is redundant?
Does kaniko skip tls verification if the registry name does not match?

Thanks
Tejal

Reply:
so kaniko can build multiple destination images if the `--destination` flag is set multiple times. it will only skip tls for the registries defined by the `--skip-tls-verify-registry` flag.

Reply:
cool. 
Thanks!